python/sparkdl/estimators/keras_image_file_estimator.py

```diff
@@ -70,6 +71,8 @@ def __next__(self):
         return self.models[index]
 
 
+@deprecated(reason="KerasImageFileEstimator will be removed in the next major Databricks Runtime "
```

Comment:
This should say "release of sparkdl" not "Databricks Runtime ML release."
Same thing elsewhere.